pkg/types/serialmarketdatastore.go

```diff
@@ -43,7 +44,7 @@ func (store *SerialMarketDataStore) Subscribe(interval types.Interval) {
 	store.Subscription = append(store.Subscription, interval)
 }
 
-func (store *SerialMarketDataStore) BindStream(ctx context.Context, stream types.Stream) {
+func (store *SerialMarketDataStore) BindStream(ctx context.Context, stream Stream) {
 	if store.UseMarketTrade {
 		if IsBackTesting {
```

Comment:
perhaps remove this condition to simplify the package import

Reply:
I'll comment out this first.
Let's see if we need to add it back in the future.